pkg/sqlcmd/sqlcmd_windows_amd64.go

```diff
@@ -7,12 +7,12 @@ import (
 )
 
 func init() {
-	if len(msdsn.ProtocolParsers) == 3 {
-		// reorder the protocol parsers to lpc->tcp->np
+	if len(msdsn.ProtocolParsers) == 4 {
```

Comment:
Can you add a comment on to why this is scoped to just amd64 on windows (per filename)?  Not ARM64?